tests/pages/project/settings/profile.js

```diff
@@ -1,17 +1,30 @@
 import {
+  clickable,
+  collection,
   create,
   text,
   visitable
 } from 'ember-cli-page-object';
 import projectSettingsForm from '../../components/project-settings-form';
+import skillsInput from '../../components/skills-input';
 
 export default create({
   visit: visitable(':organization/:project/settings/profile'),
 
+  projectSkillsList: collection({
+    scope: '.project-skills-list',
```

Comment:
This same comment would apply to the other page objects that were created, but I don't want to pollute the review with several of the same things.